lib/urbanopt/reopt/reopt_ghp_api.rb

```diff
@@ -0,0 +1,188 @@
+# *********************************************************************************
+# URBANopt (tm), Copyright (c) Alliance for Sustainable Energy, LLC.
+# See also https://github.com/urbanopt/urbanopt-reopt-gem/blob/develop/LICENSE.md
+# *********************************************************************************
+
+require 'net/https'
+require 'openssl'
+require 'uri'
+require 'json'
+require 'securerandom'
+require 'certified'
+require_relative '../../../developer_nrel_key'
+require 'urbanopt/reopt/reopt_logger'
+
+module URBANopt # :nodoc:
+  module REopt  # :nodoc:
+    class REoptLiteGHPAPI
```

Comment:
Ideally I would have liked to reuse the existing API class, but the methods for the api calls as well as the API urls are different limiting reusability.

Reply:
Those sound relatively easy to overcome, no? I'd prefer not to have 2 separate API classes.

Reply:
Id like to have this merged today. If you would like to give merging a shot feel free and I can review it.

Reply:
let's keep both classes for now since we are in a time crunch and Tanushree has already got it to work this way.  we can revisit merging the 2 together in the future.